dev-client/package.json

```diff
@@ -24,15 +25,18 @@
     "@rnmapbox/maps": "^10.1.11",
     "@sentry/react-native": "^5.18.0",
     "base-64": "^1.0.0",
-    "expo": "^50.0.6",
-    "expo-av": "^13.10.5",
-    "expo-camera": "^14.0.4",
-    "expo-screen-orientation": "^6.4.1",
-    "expo-sensors": "^12.9.1",
+    "expo": "~50.0.6",
+    "expo-av": "~13.10.5",
+    "expo-camera": "~14.0.4",
+    "expo-image-picker": "~14.7.1",
+    "expo-screen-orientation": "~6.4.1",
+    "expo-sensors": "~12.9.1",
```

Comment:
question: why do we want to use `~` instead of `^`?

Reply:
because expo makes breaking changes on minor version increments. (this is a bit reductive but) `~` means "this version and any patches to it", and `^` means "this version and any minor upgrades or patches to it". expo recommends `~` and uses it by default when you `expo install` a new package because they don't try to avoid breaking changes in minor upgrades. this was the source of e.g. this problem that snagged you & david for a while at the beginning of the month: https://tech-matters.slack.com/archives/C02K02L3Q48/p1706818388930309